packages/core/src/package-graph/__tests__/package-graph.spec.ts

```diff
@@ -189,6 +189,28 @@ describe('PackageGraph', () => {
       expect(node).toHaveProperty('localDependents', expect.any(Map));
     });
 
+    it('handles yarn patch protocol versions transparently and without hiccups', () => {
+      const pkg = new Package(
+        {
+          name: 'my-pkg',
+          version: '1.0.0',
+          dependencies: {
+            '@ionic-native/splash-screen':
+              'patch:@ionic-native/splash-screen@npm%3A5.36.0#~/.yarn/patches/@ionic-native-splash-screen-npm-5.36.0-531cbbe0f8.patch',
+          },
+        } as unknown as RawManifest,
+        '/path/to/my-pkg'
+      );
+      const node = new PackageGraph([pkg]).get('my-pkg');
+
+      console.log('node=', node);
```

Comment:
we probably don't need this `console.log`, do we? If not please remove it.

Reply:
@ghiscoding Oops, that was left there by accident, apologies. Sending an update soon!

Reply:
@ghiscoding Removed the logging, please see the updated sources!